csv-generator/tests/consumers_test/version_consumer_test.py

```diff
@@ -16,15 +16,26 @@ def test_can_get_versions(version_consumer: VersionXMLConsumer,
     assert len(version_consumer.get_versions(manuscript)) == 2
 
 
+def test_can_get_senior_editor_person_id(version_consumer: VersionXMLConsumer,
```

Comment:
minor: should we maybe have a test to check that it produces blank if there isn't a senior editor?

Reply:
Ok will add some 👍